storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.3.rst

```diff
@@ -0,0 +1,37 @@
+============================
+ |Percona XtraBackup| 2.3.3
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.3 on December 17th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/XtraBackup/xtrabackup-2.3.3/>`_. Downloads are available from our `download site <http://www.percona.com/downloads/XtraBackup/LATEST/>`_ and from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3 series.
+
+Bugs fixed
+----------
+
+ Database directories were not removed if ``DROP DATABASE`` happened during the backup. Bug fixed :bug:`1461735`.
+
+ Backup would fails if :ref:`storing_history` feature was enabled. Bug fixed :bug:`1509812`.
+
+ In case of :option:`--defaults-extra-file` option was not the first command line argument |innobackupex| would fail to authenticate and backup would fail. Bug fixed :bug:`1511451`.
+
+ Backup would fail if :option:`--rsync` option was used without specifying temporary folder. Bug fixed :bug:`1511701`.
+
+ Fixed |Percona XtraBackup| crash which happened when it was used for SST on *MariaDB Galera Cluster* caused by double free of :option:`datadir`' variable. Bug fixed :bug:`1512281`.
+
+ :option:`--move-back` did not respect the :option:`innodb_log_group_home_dir` which caused ``ib_logfiles`` not to be moved to correct location. Bug fixed :bug:`1512616`.
```

Comment:
not only innodb_log_group_home_dir, but also innodb_data_home_dir, so datafiles could be wrong location too